animation keyframes/animation.css

```diff
@@ -0,0 +1,127 @@
+div {
```

Comment:
Please do not define flex properties on global divs. Use it with classes if needed.